### PR TITLE
Adds functionality for user customized messages

### PR DESCRIPTION
**Pull request by katelovescode**
This solves #1

### CODE REVIEW

File: lib/aphorism.rb [hunk 1]
```diff
@@ -4,9 +4,23 @@ module Aphorism
...
+    def internal_aphorisms
       aphorisms_path = File.join(__dir__, 'content/aphorisms.txt')
-      aphorisms = IO.read(aphorisms_path).split("\n%\n")
-      puts aphorisms.sample
+      IO.read(aphorisms_path).split("\n%\n")
+    end
+
+    def custom_aphorisms
+      custom_path = File.join(Dir.home, '.aphorism')
+      custom_files = Dir.children(custom_path)
```
Comment (phallstrom) [inline lib/aphorism.rb:20]: If this directory doesn't exist, it will throw a Errno::ENOENT...

File: lib/aphorism.rb [hunk 2]
```diff
@@ -4,9 +4,23 @@ module Aphorism
...
+      custom_aphorisms = []
+      custom_files.each { |file| custom_aphorisms += IO.read(File.join(custom_path, file)).split("\n%\n") }
```
Comment (phallstrom) [inline lib/aphorism.rb:22]: Probably not critical, but this will pick up more than just `*.txt` files...

File: lib/aphorism.rb (hunk 1 above)
Comment (katelovescode) [reply lib/aphorism.rb:20]: Fixed in #8

File: lib/aphorism.rb (hunk 2 above)
Comment (katelovescode) [reply lib/aphorism.rb:22]: Fixed in #8

**Pull request merged**